src/server/debug/server/server.go

```diff
@@ -689,13 +788,14 @@ func (s *debugServer) collectCommits(rctx context.Context, tw *tar.Writer, pachC
 			}
 			return errors.EnsureStack(s.marshaller.Marshal(w, ci))
 		})
-	}, prefix...); err != nil {
+	}); err != nil {
 		return err
 	}
 	// Reverse the x values since we collect them from newest to oldest.
 	// TODO: It would probably be better to support listing jobs in reverse order.
```

Comment:
This comment talks about listing jobs in reverse order, which we can do, but this function is listing commits, which we can't do in reverse order 😂